### PR TITLE
Feature/support new crypotbox

### PR DESCRIPTION
**Pull request by typfel**
The cryptobox API has changed which requires the mock transport to be updated.

### CODE REVIEW

File: Cartfile.resolved [hunk 1]
```diff
@@ -1,9 +1,9 @@
-github "wireapp/ocmock" "v3.2.2"
```
Comment (marcoconti83) [inline Cartfile.resolved]: Why this change for ocmock?

Comment (typfel) [inline Cartfile.resolved]: that's what's happened when I did carthage update

Comment (marcoconti83) [inline Cartfile.resolved]: Interesting. I would revert it, let's aim to keep compatibility with v3 in all frameworks, and one day we will migrate to v3 on sync-engine too

Comment (typfel) [inline Cartfile.resolved]: Ok
